torch/csrc/jit/script/module.h

```diff
@@ -328,6 +328,92 @@ struct TORCH_API Module {
 
     return nullptr;
   }
+
+  void validateGetSetStateSchema() const {
```

Comment:
I am not a fan of how long this error checking code is, but I don't have a better solution for you.

Reply:
It'd be nice if we could specify a target schema string to match to, but that's outside the scope of this PR.